chroma-manager/Makefile

```diff
@@ -79,14 +78,9 @@ develop: version ui-modules
 	python setup.py develop
 	./manage.py dev_setup $(DEV_SETUP_BUNDLES)
 
-$(HOME)/.node-gyp:
-	@echo "copying nodejs headers"; \
-	mkdir -p $HOME/.node-gyp && \
-	cp -r ../chroma-externals/node-gyp-headers/ $(HOME)/.node-gyp/
-
-npm: $(HOME)/.node-gyp
+npm:
 ifneq ($(NPM), $(NPM_VERSION))
-	curl https://ubit-artifactory-or.intel.com/artifactory/api/npm/iml-npm-repos/npm/-/npm-$(NPM_VERSION).tgz > /tmp/npm-$(NPM_VERSION).tgz
+	curl https://registry.npmjs.org/npm/-/npm-$(NPM_VERSION).tgz > /tmp/npm-$(NPM_VERSION).tgz
```

Comment:
Node should (does) provide npm by default. This is only here because of the bridge dep, once we upgrade to latest epel, we can remove this whole thing.